utils/buildJsonResults.js

```diff
@@ -64,11 +65,7 @@ module.exports = function (report, appDirectory, options) {
       if (tc.status === 'failed') {
         tc.failureMessages.forEach((failure) => {
           testCase.testcase.push({
-            'failure': [{
-              _attr: {
-                message: failure || ''
-              }
-            }]
+            'failure': stripAnsi(failure || '')
```

Comment:
The "_attr" attribute needs to be there otherwise the format of the junit.xml won't be to spec